Makefile

```diff
@@ -171,8 +171,8 @@ $(FIRECRACKER_CONTAINERD_RUNTIME_DIR) $(ETC_CONTAINERD):
 DEFAULT_VMLINUX_NAME?=default-vmlinux.bin
 $(DEFAULT_VMLINUX_NAME):
 	curl --silent --show-error --retry 3 --max-time 30 --output $@ \
-		"https://s3.amazonaws.com/spec.ccfc.min/img/hello/kernel/hello-vmlinux.bin"
-	echo "882fa465c43ab7d92e31bd4167da3ad6a82cb9230f9b0016176df597c6014cef $@" | sha256sum -c -
+		"https://s3.amazonaws.com/spec.ccfc.min/img/quickstart_guide/x86_64/kernels/vmlinux.bin"
```

Comment:
out of curiosity, where did we get this path from?

Reply:
https://github.com/firecracker-microvm/firecracker/blob/main/docs/getting-started.md#running-firecracker